src/ontology/mondo-edit.obo

```diff
@@ -27605,7 +27605,7 @@ property_value: seeAlso https://rarediseases.info.nih.gov/diseases/12803/phaeohy
 [Term]
 id: MONDO:0001868
 name: primary angle-closure glaucoma
-def: "A form of glaucoma in which there is no visible abnormality in the trabecular meshwork." [NCIT:C35394]
+def: "The sudden increase of intraocular pressure, resulting in pain and an abrupt decrease in visual acuity." [NCIT:C34639]
```

Comment:
This is better as it is unique but it does not conform to our text definition guidelines